Framing/src/authenticate_plugin.py

```
@@ -19,54 +19,31 @@
-        raise FileNotFoundError(
-            f"AuthDefaultKey.h not found at {path}. "
-            "Authentication plugin requires AuthDefaultKey.h to be present. "
-            "Ensure the file exists or run 'make generate-auth-key' to create it."
+    key = os.environ.get("PROVES_AUTH_KEY")
+    if not key:
+        raise ValueError(
+            "No authentication key available: pass --authentication-key or set "
+            "the PROVES_AUTH_KEY environment variable. The key is provisioned "
+            "onto the satellite with the PROVISION_KEY command and is never "
+            "compiled into the flight image."
         )
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)